example.config.yml

```diff
@@ -64,11 +64,38 @@ configure_local_drush_aliases: true
 
 # Apache VirtualHosts. Add one for each site you are running inside the VM. For
 # multisite deployments, you can point multiple servernames at one documentroot.
+# NOTE: We create separate vhosts for www and non-www URLs so that the
+#       hostsupdater plugin will create host file entries for both.
+# NOTE: We can't create a vhost with a servername of  "www.{{ drupal_domain }}"
+#       because the autohost plugin won't pick up names that have {{ in them.
 apache_vhosts:
   - {servername: "{{ drupal_domain }}", documentroot: "{{ drupal_core_path }}"}
+  - {servername: "www.drupalvm.dev", documentroot: "{{ drupal_core_path }}"}
   - {servername: "xhprof.drupalvm.dev", documentroot: "/usr/share/php/xhprof_html"}
   - {servername: "pimpmylog.drupalvm.dev", documentroot: "/usr/share/php/pimpmylog"}
 
+# Optionally, enable/disable additional apache modules.
+# By default, all modules required by Drupal are enabled, so only
+# Additional modules need to be listed here.
+apache_mods_enabled:
+  - expires.load
+  - ssl.load
+  - rewrite.load
+# apache_mods_disabled: []
+
+# Enable SSL Support on the Drupal site vhosts.
```

Comment:
Can you file a separate PR for the SSL vhosts (and remove it from this PR)? I want to make sure that this works correctly across different OSes and provides a seamless experience out of the box, and I'd rather tackle that separately.